include/libswiftnav/constants.h

```diff
@@ -56,10 +59,6 @@
  * \note This is GPS_C / mu where mu is 1.0002926 */
 #define GPS_C_NO_VAC (GPS_C / 1.0002926)
 
-/** The wavelength of L1 in a vacuum.
- * \note This is GPS_C / GPS_L1_HZ. */
-#define GPS_L1_LAMBDA (GPS_C / GPS_L1_HZ)
```

Comment:
This was removed to avoid implementing one more utility function like code_to_carr_freq. There is only place (observation.c) where the macro is used so I simple changed it with GPS_C/code_to_carr_freq(e[i]->sid.code) see below.